lib/gollum/app.rb

```diff
@@ -450,6 +450,7 @@ def wiki_new
       wiki         = Gollum::Wiki.new(settings.gollum_path, wiki_options)
       @results     = wiki.pages
       @results     += wiki.files if settings.wiki_options[:show_all]
+      @results     = @results.sort_by { |p| p.name.downcase } # Sort Results alphabetically, fixes 922
```

Comment:
I expected that `sort_by!` would be slightly faster, but I'm actually seeing the opposite when benchmarking. So, whatever. :)